bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -226,55 +234,48 @@ max_bestnr, _, full_time_veto_max_bestnr =\
 if onsource_file:
 
     # Get trigs
-    on_trigs = ppu.load_xml_table(onsource_file, "multi_inspiral")
-    logging.info("%d onsource triggers loaded.", len(on_trigs))
+    on_trigs = ppu.load_triggers(onsource_file, vetoes)
+    logging.info("%d onsource triggers loaded.", on_trigs['network/event_id'].len())
 
-    # Separate off chirp mass column
-    on_mchirp = on_trigs.get_column('mchirp')
+    # Calculate chirp mass values
+    on_mchirp = mchirp_from_mass1_mass2(bank_file['mass1'][:][on_trigs['network/template_id']],
+                                        bank_file['mass2'][:][on_trigs['network/template_id']])
```

Comment:
You have very similar code in a couple places, so I suggest creating a short helper function that lets you to do
```python
on_mchirp = get_trig_mchirp_from_bank(bank_file, on_trigs)
```

Reply:
Alternatively, since the only use of the bank file is to get the chirp masses, create the array of template chirp masses first with
```python
logging.info('Reading template chirp masses')
with h5py.File(opts.bank_file, 'r') as bank_file:
    template_mchirps = mchirp_from_mass1_mass2(
        bank_file['mass1'][:],
        bank_file['mass2'][:]
    )
```
and then just carry `template_mchirps` around. This lets you get rid of the bank file right away (better not keep open file handlers around).